interface/billing/print_daysheet_report_num2.php

```diff
@@ -435,29 +435,29 @@
          print  "</span></td><td width=250><span class=text><center>" . xlt('Insurance Payment'). "</center>";
      }
      if (($iter{'pat_code'}) > 0 AND ($iter{'code_type'}) === 'Patient Payment' AND $iter{'paytype'} != 'PCP' ){
-         print  "</span></td><td width=250><span class=text><center>" . xlt('Patient Payment'). "</center>";
+         print  "</span></td><td width=250><span class=text><center>" . xlt('Patient Payment')." / " . text($iter{'pay_method'}). "</center>";
      }
      if (($iter{'ins_code'}) < 0 AND ($iter{'code_type'}) === 'Insurance Payment' ){
          print  "</span></td><td width=250><span class=text><center>" . xlt('Insurance Credit'). "</center>";
      }
      if (($iter{'pat_code'}) < 0 AND ($iter{'code_type'}) === 'Patient Payment' AND $iter{'paytype'} != 'PCP' ){
          print  "</span></td><td width=250><span class=text><center>" . xlt('Patient Credit'). "</center>";
      }
-     if ($iter{'paytype'} === 'PCP') { 
-      print  "</span></td><td width=250><span class=text><center>" . xlt('COPAY'). "</center>";
+     if ($iter{'paytype'} === 'PCP') {
+      print  "</span></td><td width=250><span class=text><center>" . xlt('COPAY')." / " . text($iter{'pay_method'}). "</center>";
      }
      if (($iter{'code_type'}) != 'Insurance Payment' AND ($iter{'code_type'}) != 'Patient Payment' AND $iter{'paytype'} != 'PCP') {
          print  "</span></td><td width=100><span class=text><center>" . text($iter{'code_type'}). "</center>";
      }
-      print  "</span></td><td width=100><span class=text><center>" . text($iter{'provider_id'}). "</center>"; 
+      print  "</span></td><td width=100><span class=text><center>" . text($iter{'enc_provider_id'}). "</center>";
```

Comment:
Be good to have somewhere to configure the report for what field to check.  Thinking of referring, supervising, and interns.